libs/components/theme/src/lib/styles/themes/modern/_switch.scss

```diff
@@ -35,6 +35,10 @@
   display: inline-flex;
   position: relative;
 
+  &:has(~ .sky-control-help-container) .sky-switch-label {
```

Comment:
This is to allow the use of the help container with switch labels without breaking existing spacing.